project/src/main/java/com/google/sps/data/VolunteeringOpportunity.java

```diff
@@ -2,25 +2,29 @@
 
 import java.util.HashSet;
 import java.util.Set;
+import java.util.UUID;
 
 public final class VolunteeringOpportunity {
-  private final long opportunityId =
-    10000000L; // TODO: Needed for storing ID from DB, replace using ID generated by DB
+  private String opportunityId;
+  private String eventId;
```

Comment:
idea, maybe eventId can be replaced with the Event itself in order to lower the number of DB calls (so you don't have to query for other event fields and instead have it all in the object)

Reply:
yes, ill think about this - I am leaning towards using using an join between events and volunteering opportunities table.